### ISSUE TITLE
Error Accessing FTP from Other Computer

### ISSUE THREAD

**Issue by Mvhin12** [jamiegau/cinema-catcher-app#13]
![Uploading IMG20241231154718.jpg…]()
Hi! This project is very helpfull to me as I am using this as a storage of our dcp. I am having an issue accessing the ftp using the account media:media and upload:upload account indicated in the Media Asset Tab. Can you help me on this please? I attached the error message to this ticket.

**Comment by jamiegau** [comment]
The image did not work.
Please describe the issue.

**Comment by Mvhin12** [comment]
I am having issue with ftp connection with message:

Connection failed.
OOPS: cannot change
directory:/opt/catcher/storage/assetlib/by_type

Tried to access the cinema catcher via ftp with username and password media:media and upload:upload but cannot connect.

**Comment by Mvhin12** [comment]
![catcher](https://github.com/user-attachments/assets/2a98e45f-fdb4-4fac-b94d-650850ca9a43)
see error message

**Comment by jamiegau** [comment]
This error would lead me to believe you didn;t follow the installation correctly.
On the docker server, did you make the directory
```
/opt/catcher/storage
```
Before you started the containers?

**Comment by Mvhin12** [comment]
Yes, i created a directory to /opt/catcher/storage and mount my harddrive to that directory. This is the 1st instruction to to create a diretcory to store all filles. I succesfully ingested a dcp from a usb and found the filea to that directory. The only problem is that i cant access that directory using ftp from different computer and also from a doremi showvault when adding it to Media source.

**Comment by jamiegau** [comment]
I don't know what to say.
When the docker containers start, if any directories under /opt/catcher/storage do not exist, they should be created automatically.

I would suggest doing a 
```
docker compose down
then
docker compose up -d
```
To see if it helps.

Otherwise, look for errors in the docker compose output when you start the containers.

**Comment by Mvhin12** [comment]
Maybe I can just add another ftp user account and set the home directory to /opt/catcher/storage.
can you send guides on how to add ftp account?

**Comment by Mvhin12** [comment]
Solved already. This error was caused by directory permission of Ubuntu and UFW firewall.
